packages/web-frontend/src/sagas.js

```diff
@@ -529,6 +529,7 @@ function* attemptRequestCountryAccess(action) {
   try {
     yield call(request, requestResourceUrl, fetchRequestCountryAccessError, options);
     yield put(fetchRequestCountryAccessSuccess());
+    yield call(fetchProjectData);
```

Comment:
@rohan-bes or @edmofro, just wanted a second opinion on this change, seems to me the easiest way to refresh the project access on the project page after request.

Reply:
Yeah this seems fine to me! I'm not very familiar with the sagas file though.